### PR TITLE
Aerogear 2440

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation
 
Upstream version of https://github.com/aerogear/aerogear-js-sdk/pull/45

### CODE REVIEW

File: packages/auth/package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  },
+  "devDependencies": {
+    "@types/bluebird": "3.5.20",
+    "@types/chai": "4.1.2",
+    "@types/chai-as-promised": "7.1.0",
+    "@types/mocha": "^5.0.0",
+    "@types/node": "^9.4.6",
+    "@types/proxyquire": "1.3.28",
+    "@types/sinon": "^4.3.1",
+    "bluebird": "^3.5.1",
```
Comment (ciaranRoche) [inline packages/auth/package.json]: Do we still need this dependency?

**Comment by wtrocki** [comment]
@ciaranRoche We still need bluebird. Going to add stuff to this PR.

**Comment by wtrocki** [comment]
@ciaranRoche @StephenCoady Reviewed. Can we approve that?

File: packages/auth/package.json (hunk 1 above)
Comment (wtrocki) [reply packages/auth/package.json]: Yes. Temp workaround.

**Pull request merged**